trade-finance-manager-api/src/v1/controllers/party.controller.js

```diff
@@ -0,0 +1,34 @@
+const api = require('../api');
+const { canDealBeSubmittedToACBS, submitACBSIfAllPartiesHaveUrn } = require('./deal.controller');
+
+const updateParty = async (req, res) => {
+  const { dealId } = req.params;
+  const partyUpdate = {
+    tfm: {
+      parties: req.body,
+    },
+  };
+
+  try {
+    const updatedDeal = await api.updateDeal(dealId, partyUpdate);
+
+    if (updatedDeal.dealSnapshot) {
+      if (await canDealBeSubmittedToACBS(updatedDeal.dealSnapshot.submissionType)) {
```

Comment:
I'm guessing was just copy pasted but I think this isn't an async function and doesn't need to be awaited